Mad.Api/Mad.Api/Controllers/WeatherForecastController.cs

```diff
@@ -48,5 +48,20 @@ public IActionResult DeleteCurrentWeatherTest()
             return Ok();
         }
 
+        [HttpPost("post-current")]
+
+        public string NotificationToCivilians()
+        {
+            var NowWeather = new WeatherForecast();
+            NowWeather.TemperatureC = Random.Shared.Next(-45, 55);
+            string alert = $"Everything is nice! Enjoy your life! The Temperature is {NowWeather.TemperatureC}";
+            if (NowWeather.TemperatureC > 25 || NowWeather.TemperatureC < -5)
```

Comment:
```suggestion
            if (NowWeather.TemperatureC is > 25 or < -5)
```
You correct, but you can do better!